optimizely/odp/odp_event_manager.py

```diff
@@ -40,20 +40,23 @@ class OdpEventManager:
     The OdpEventManager maintains a single consumer thread that pulls events off of
     the queue and buffers them before events are sent to ODP.
     Sends events when the batch size is met or when the flush timeout has elapsed.
+    Flushes the event queue after specified time (seconds).
     """
 
     def __init__(
         self,
         logger: Optional[_logging.Logger] = None,
         api_manager: Optional[OdpEventApiManager] = None,
-        timeout: Optional[int] = None
+        timeout: Optional[int] = None,
```

Comment:
I think that will help clarify the difference:
```suggestion
        request_timeout: Optional[int] = None,
```

Reply:
done